parachain/src/chain_spec/rococo.rs

```diff
@@ -61,7 +26,7 @@ where
 
 pub fn get_chain_spec(para_id: ParaId) -> ChainSpec {
 	let mut props = Properties::new();
-	props.insert("tokenSymbol".into(), "DEV".into());
+	props.insert("tokenSymbol".into(), "ROC".into());
```

Comment:
is this creating an additional ROC token on our parachain? or refering to the relay chain's token? i would rather call it SNOW or smth if its a new token